picard/script/functions.py

```diff
@@ -286,6 +286,30 @@ def func_replace(parser, text, old, new):
     return text.replace(old, new)
 
 
+@script_function(eval_args=False, documentation=N_(
+    """`$replacemulti(value,search,replace,separator="; ")`
```

Comment:
I suggest changing this to:

```
    """`$replacemulti(name,search,replace,separator="; ")`
```

to make it consistent with the documentation on Line 292.

Reply:
I took the freedom to quickly fix this naming, will merge this now. Thanks a lot @atj